### PR TITLE
SE-2946: Create sample notebook to demonstrate TxnConfig side can be set up to change instrument units

### PR DESCRIPTION
**Pull request by ghost**
# Pull Request Checklist

- [x] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [x] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

File: examples/features/core-lusid/Transaction Configuration.ipynb [hunk 1]
```diff
@@ -0,0 +1,2193 @@
+{
```
Comment (Tomwp) [inline examples/features/core-lusid/Transaction Configuration.ipynb:1]: <ol><li>I think we should give a more detailed overview of how typical security purchases draw money out of the LUSID cash security, whereas this example shows how money can be drawn out of a reserve fund modelled as a security.</li><li>Should put attributes in a doc string.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/648/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (Tomwp) [inline examples/features/core-lusid/Transaction Configuration.ipynb:1]: <p>Can we put the first set of bullets into a table of contents?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/648/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (Tomwp) [inline examples/features/core-lusid/Transaction Configuration.ipynb:1]: <p>You should mention that "Buy" is a default type that gets shipped with LUSID and mention that this is the typical way a purchase is booked.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/648/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (Tomwp) [inline examples/features/core-lusid/Transaction Configuration.ipynb:1]: <ul><li>We should rename the columns into a more user friendly / readable format. </li><li>Don't need all of these columns. We can probably remove cost (in portfolio currency) columns, holding type, subholding keys, effective from, holding type, and settled units</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/648/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (Tomwp) [inline examples/features/core-lusid/Transaction Configuration.ipynb:1]: <ul><li>We should rename the columns into a more user friendly / readable format. </li><li>Don't need all of these columns. We can probably remove cost (in portfolio currency) columns, holding type, subholding keys, effective from, holding type, and settled units</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/648/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (Tomwp) [inline examples/features/core-lusid/Transaction Configuration.ipynb:1]: <ul><li>We should rename the columns into a more user friendly / readable format. </li><li>Don't need all of these columns. We can probably remove cost (in portfolio currency) columns, holding type, subholding keys, effective from, holding type, and settled units</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/648/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Comment (Tomwp) [inline examples/features/core-lusid/Transaction Configuration.ipynb:1]: <p>I think we want to avoid showing any USD cash lines. For the USD instrument, we should just book against the reserve fund. This will show a clear distinction between booking against the cash method (i.e. GBP).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/648/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (Tomwp) [inline examples/features/core-lusid/Transaction Configuration.ipynb:1]: <ul><li>We should rename the columns into a more user friendly / readable format. </li><li>Don't need all of these columns. We can probably remove cost (in portfolio currency) columns, holding type, subholding keys, effective from, holding type, and settled units</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/648/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (Tomwp) [inline examples/features/core-lusid/Transaction Configuration.ipynb:1]: <ul><li>We should rename the columns into a more user friendly / readable format. </li><li>Don't need all of these columns. We can probably remove cost (in portfolio currency) columns, holding type, subholding keys, effective from, holding type, and settled units</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/648/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

File: examples/features/core-lusid/Transaction Configuration.ipynb [hunk 2]
```diff
@@ -0,0 +1,1406 @@
...
```
Comment (Tomwp) [inline examples/features/core-lusid/Transaction Configuration.ipynb:1]: <p>I think we should add an explanation of this specific use case (i.e. we're drawing cash out of a reserve instrument rather than the standard cash instrument).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/648/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (Tomwp) [inline examples/features/core-lusid/Transaction Configuration.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #5.    Shows transaction configuration created in LUSID by default, and how to create customised transaction types in transaction configuration. 
</pre><p>Can remove 'transaction configuration' at the end of the sentence.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/648/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (Tomwp) [inline examples/features/core-lusid/Transaction Configuration.ipynb:1]: <p>First paragraph:</p><p> </p><ul><li>I would reword the first sentence: "<a href="https://support.lusid.com/knowledgebase/article/KA-01872" target="_blank">Transaction Configuration</a> is a core feature of LUSID that provides a great deal of flexibility..."</li><li>Remove 'also' in second sentence.</li></ul><p> </p><p>Second paragraph:</p><p> </p><p>I would reword this as follows: <strong><em>In this notebook, we demonstrate how to create a custom buy transaction that funds the security purchase out of a cash reserve fund rather than the default LUSID currency instrument.</em></strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/648/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (Tomwp) [inline examples/features/core-lusid/Transaction Configuration.ipynb:1]: <p>I would comma separate these words rather than using slashes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/648/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (Tomwp) [inline examples/features/core-lusid/Transaction Configuration.ipynb:1]: <ul><li>First sentence I would just say: "Upsert 3 transactions into our portfolio"</li><li>I would say 'deposit' or 'inject' rather than 'put'.</li><li>Say 'our portfolio' or 'the portfolio'</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/648/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

[the remark this answers is not in the document]
Comment (Yemmy-At-Finbourne) [reply examples/features/core-lusid/Transaction Configuration.ipynb:1]: <p>they should be all fixed now. </p>
